shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
The only way to vary the dependencies is through Gemfiles though as far as I know. Travis (or any CI) doesn't let you specify the gemspec file. Is what where you were recommending? That would be the ideal solution if it was possible.

Here's another example of how we have it setup: https://github.com/Shopify/active_fulfillment

* Dependencies in gemspec
* Default Gemfile only refers to gemspec
* Other Gemfiles only override what they need to